src/python/tools/compiler_utils.py

```diff
@@ -120,13 +120,19 @@ def _nodes_with_presteps(self):
     '''from ..schema import TypeConverter
 return [TypeConverter(result_type='R4')._steal_io(self), self]'''
 
+timeseries_to_r4_converter = \
+    '''from ..preprocessing.schema import TypeConverter
+return [TypeConverter(result_type='R4')._steal_io(self), self]'''
+
```

Comment:
You can reuse int_to_r4_converter instead of duplicating

Reply:
`int_to_r4_converter` does not work correctly because of the `from .schema` line which is not accessible from the `time_series` folder.